js/components/interface/listViewer/showcase/ListViewerShowcase.js

```diff
@@ -0,0 +1,191 @@
+import React from 'react';
```

Comment:
@filippomc is there a page which uses the showcase?

Reply:
No, I was planning to add it to Geppetto Application but I'm not sure anymore it's the right place. Too bound with the Java backend and the classic examples. What about having the showcase in geppetto-client itself or a separate repo?

Reply:
Well geppetto-application is deployed to live so there's that advantage if we put it there. Being coupled with a backend also gives us projects with real instances which might be useful to showcase some components. The thing is if you want to put it online there has to be some kind of backend, imagine showcasing a component which takes advantage of the model or of the api for instance, it's not just a matter like in material ui to serve some static files maybe. Thoughts?

Reply:
I'm not convinced that the showcaser and the user must be aware of a specific project content and have the whole java stack up to setup locally. Geppetto live is ok to showcase the neuroscience capabilities of geppetto, I don't see how to make that become a guide for the developer. Yes we need the instances for some components, but nothing we can't mock.

Reply:
I think we need to write down what goals we have for the showcase and what we want it to look like, I'm starting to sense different pictures being painted in different brains.

Reply:
Agree